shared_tasks/dynalab-base-nli/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.8
+COPY ./app ./app
```

Comment:
Can you remove the Dynabench 2.0 instructions this is quite confusing.

Reply:
I removed all the dynabench 2.0 stuff